tests/checkpoint/test_checkpoint.py

```diff
@@ -1150,37 +1140,50 @@ def test_checkpoint_run_with_slack_action_no_page_links(self, tmp_path: pathlib.
 @pytest.mark.unit
 def test_is_fresh(
     in_memory_runtime_context,
-    id: str | None,
-    validation_def_id: str | None,
-    suite_id: str | None,
-    batch_def_id: str | None,
-    is_fresh: bool,
+    id: bool,
```

Comment:
Unfortunately not because overwriting an ID with a new UUID causes things to be out of sync (which is now important because we're retrieving with stores)

Reply:
Ah, gotcha. Makes sense!